x-pack/platform/plugins/shared/cases/public/components/case_view/components/case_view_alerts.tsx

```diff
@@ -86,6 +86,7 @@ export const CaseViewAlerts = ({
             ? SECURITY_SOLUTION_RULE_TYPE_IDS
             : alertData?.ruleTypeIds ?? []
         }
+        caseId={caseData.id}
```

Comment:
I couldn't believe the AlertsTable wasn't being given access to the active case ID, I had to add it like this. I think this causes a console error about this value being added to a <div> somewhere down the line.